### ISSUE TITLE
Clean up "cur chuige"

### ISSUE THREAD

**Issue by kscanne** [UniversalDependencies/UD_Irish-IDT#87]
Sometimes "chuige" is linked to "cur" with fixed, sometimes as an obl:prep, and other times it is annotated as "case" of a following noun which is definitely wrong.  I'd be in favor of obl:prep but open to other solutions.

**Comment by tlynn747** [comment]
obl:prep is fine as a solution, but should "cur" perhaps be POS-tagged as a verbal noun?

**Comment by kscanne** [comment]
It's a strange hybrid, a bit like "cur síos".  I see it used in the plural quite a bit online which makes it seem less like a verbal noun (sometimes "na cuir chuige", more often "na cur chuigí", which opens a whole new set of questions!).  Is the idea that you'd want nmod instead of obl if it's a regular noun?

**Comment by tlynn747** [comment]
yes that was the motivation for the proposal :) obl-s are usually only attached to verbs. 
Is na "cuir chuige" found in UGC as opposed to standard text? If it is, then I wouldn't see it as a motivation for keeping it as NOUN in general.
In that vein - what would the plural of "cur síos" be, or is there a legit plural?

**Comment by kscanne** [comment]
I see "na cuir chuige" is some government docs, including one from Roinn na Gaeltachta!  It's also given some legitimacy by tearma.ie: https://www.tearma.ie/q/cur%20chuige/ 
There's more evidence of non-verbal-noun-ness even in the treebank examples, where it's modified by adjectives, e.g. "cur chuige éiceachórasach" or preceded by the definite article: "athbhreithniú ar an gcur chuige forbartha" etc.

Dictionaries/téarma don't give a plural of "cur síos". I only see one "na cuir síos" in my corpus.

**Comment by tlynn747** [comment]
On board with the evidence of non-verbal-noun-ness!  Do we have a precedent for obl being attached to nouns in exceptional cases like this?

**Comment by kscanne** [comment]
Not really — I'd go with nmod since "chuige" is really like any other PP.

**Comment by tlynn747** [comment]
OK grand - let's go with nmod then

**Comment by michealjohnny** [comment]
> 
> 
> I see "na cuir chuige" is some government docs, including one from Roinn na Gaeltachta! It's also given some legitimacy by tearma.ie: https://www.tearma.ie/q/cur%20chuige/
> There's more evidence of non-verbal-noun-ness even in the treebank examples, where it's modified by adjectives, e.g. "cur chuige éiceachórasach" or preceded by the definite article: "athbhreithniú ar an gcur chuige forbartha" etc.
> 
> Dictionaries/téarma don't give a plural of "cur síos". I only see one "na cuir síos" in my corpus.

Give the plural time! "Cur síosanna" has been increasing in frequency.
